WowCharacterComparer.Tests/Logic/User/PasswordValidationManagerTest.cs

```diff
@@ -1,90 +1,165 @@
 ﻿using WowCharComparerWebApp.Configuration;
-using WowCharComparerWebApp.Logic.User;
 using Xunit;
 using System.Linq;
+using WowCharComparerWebApp.Logic.Users;
 
 namespace WowCharacterComparer.Test
 {
     public class PasswordValidationManagerTest
     {
-        [Theory]
-        [InlineData("Infermus123@vp.pl", true, "")]
-        [InlineData("Infermus123%vp.pl", false, UserMessages.UserEmailInvalidFormat)]
-        public void CheckUserEmail_WhenVariousUserInput_ShouldReturnExpectedBehaviour(string userEmail, bool status, string message)
+        [Fact]
+        public void CheckUserEmail_WhenUserInputIsCorrect_ShouldReturnCorrectResult()
         {
             //Arrange 
-            (bool, string) expected = (status, message);
+            (bool, string) expected = (true, string.Empty);
+            string userEmail = "Infermus123@vp.pl";
 
             //Act
-            var result = new PasswordValidationManager(null).CheckEmail(userEmail);
+            var result = new PasswordValidationManager(null, null).CheckEmail(userEmail);
 
             //Assert
             Assert.Equal(expected, result);
         }
 
-        [Theory]
-        [InlineData("TestUser", true, "")]
-        [InlineData("TestU", false, UserMessages.UserNameLengthTooShort)]
-        public void CheckUserName_WhenVariousUserInput_ShouldReturnExpectedBehaviour(string userName, bool status, string message)
+        [Fact]
+        public void CheckUserEmail_WhenUserInputIsInvalid_ShouldReturnErrorWithSuitableMessage()
```

Comment:
If you really gonna create every test method for every invalid case, better use more accurate second part of method name. Here for example we can go for:
`CheckUserEmail_WhenUserEmailIsInvalid_ShouldReturnErrorWithSuitableMessage()`
Same rule for rest of method.

Current name fits when you use "theory" (Xunit) for all included in this file cases.